ex2.sh

```diff
@@ -0,0 +1,40 @@
+# question 1:
+
+ls -S
+
+# a : 
+# a lot of text: can find with cat or less (in our case is : cat /etc/services)
+
+# b:  
+# first i found the biggest file
+ ls -S /etc/ | head -1
+# then, i can show the contact of the file:
+cat /etc/services
+
+# question 2:
+ls -S -r

```

Comment:
The command is correct, but you can combine the flags -S and -r to simplify the command